examples/basic_website_jsx/Readme.md

```diff
@@ -0,0 +1,13 @@
+This is a very simple website in pure tyxml using the jsx syntax extension.
+To generate the website, compile `site_html.ml` and then execute. This can be done with `make`.
+
+Content of this directory:
+- `site_html.ml`: Generates the Html.
+- `Makefile`: Simple rules to create the website. Uses ocamlbuild
+- `main.js` and `home.css` : auxiliary files for the website.
+- `.merlin`: An appropriate merlin file.
```

Comment:
this file doesn't exist

Reply:
I think these are generated in all the examples, not sure if/how we want to address this?

Reply:
it's very probably legacy from the before dune days